commit 783d920438db8e86d394363d824076ca45214161
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Fri Mar 17 00:48:24 2017 +0000

    PCI: Ignore BAR updates on virtual functions
    
    [ Upstream commit 63880b230a4af502c56dde3d4588634c70c66006 ]
    
    VF BARs are read-only zero, so updating VF BARs will not have any effect.
    See the SR-IOV spec r1.1, sec 3.4.1.11.
    
    We already ignore these updates because of 70675e0b6a1a ("PCI: Don't try to
    restore VF BARs"); this merely restructures it slightly to make it easier
    to split updates for standard and SR-IOV BARs.
    
    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
    Reviewed-by: Gavin Shan <gwshan@linux.vnet.ibm.com>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1429_783d920438db8e86d394363d824076ca45214161.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1429_783d920438db8e86d394363d824076ca45214161.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
